pkg/kudoctl/cmd/init_integration_test.go

```diff
@@ -41,6 +44,86 @@ func TestMain(m *testing.M) {
 	os.Exit(exitCode)
 }
 
+const (
+	operatorFileName        = "kudo_v1alpha1_operator.yaml"
+	operatorVersionFileName = "kudo_v1alpha1_operatorversion.yaml"
+	instanceFileName        = "kudo_v1alpha1_instance.yaml"
+	manifestsDir            = "../../../config/crds/"
+)
+
+func TestCrds_Config(t *testing.T) {
+	crds := cmdinit.CRDs()
+
+	if false {
+		// change this to true if you want to one time override the manifests with new values
```

Comment:
Wouldn't it be nice to have a `make test-crds` which would call into a `main()` method (can be in this test file) that writes the manifests? Instead of manually changing `false -> true` here? Just an idea

Reply:
I'll leave that as an exercise for someone who is not comfortable with the true/false situation 😛